turbo/mixins.py

```diff
@@ -21,15 +33,20 @@ def _broadcast_to_instance(self, instance, action):
                 "pk": self.pk,
                 "action": action,
                 "channel_name": channel_name,
+                "template": partial,
+                "context": context,
             })
 
-    def save(self, *args, **kwargs):
-        action = self.inserts_by if self._state.adding else "replace"
-        super().save(*args, **kwargs)
+    def broadcast(self, created):
+        action = self.inserts_by if created else "replace"
```

Comment:
Is there a way to support removes too?

Reply:
Not yet... I can work on adding remove support, though, through an override on the `delete()` method